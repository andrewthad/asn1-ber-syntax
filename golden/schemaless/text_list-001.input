300A0C03 6161610C 03626262
