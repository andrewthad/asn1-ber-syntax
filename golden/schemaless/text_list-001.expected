Value
    { tagClass = Universal
    , tagNumber = 16
    , contents = Constructed
        ( fromListN 2
            [ Value
                { tagClass = Universal
                , tagNumber = 12
                , contents = Primitive
                    [ 0x61
                    , 0x61
                    , 0x61
                    ]
                }
            , Value
                { tagClass = Universal
                , tagNumber = 12
                , contents = Primitive
                    [ 0x62
                    , 0x62
                    , 0x62
                    ]
                }
            ]
        )
    }
